client/state/data-layer/wpcom/sites/rewind/schema.js

```diff
@@ -0,0 +1,102 @@
+/** @format */
+
+export const credential = {
+	type: 'object',
+	properties: {
+		still_valid: { type: 'boolean' },
+		type: { type: 'string', enum: [ 'auto', 'ftp', 'sftp', 'ssh' ] },
+		host: { type: 'string' },
+		port: { type: 'integer' },
+	},
+	required: [ 'still_valid', 'type' ],
+};
+
+export const download = {
+	type: 'object',
+	properties: {
+		downloadId: { type: 'integer' },
+		rewindId: { type: 'string' },
+		backupPoint: { type: 'integer' },
+		startedAt: { type: 'integer' },
+		downloadCount: { type: 'integer' },
+		validUntil: { type: 'integer' },
+	},
+	required: [ 'downloadId', 'rewindId', 'backupPoint', 'startedAt' ],
+};
+
+export const restore = {
+	type: 'object',
+	properties: {
+		status: { type: 'string', enum: [ 'inactive', 'queued', 'running' ] },
```

Comment:
Does this correspond to the following? I'd expect the lists to match.

https://github.com/Automattic/wp-calypso/blob/ec36c6f3973ce8dd976745e5a670ba20a62fc181/client/my-sites/stats/activity-log/index.jsx#L205-L216

Reply:
somewhat aligned in 0af4494e8